ChronoAPI/ChronoLog/include/ChronoLogAdminRPCVisor.h

```diff
@@ -16,11 +16,12 @@
 #include "ClientRegistryInfo.h"
 #include "ClientRegistryManager.h"
 
-class ChronoLogAdminRPCProxy {
+class ChronoLogAdminRPCVisor {
 public:
-    ChronoLogAdminRPCProxy() {
+    ChronoLogAdminRPCVisor() {
         LOGD("%s constructor is called", typeid(*this).name());
-        rpc = ChronoLog::Singleton<ChronoLogRPCFactory>::GetInstance()->GetRPC(CHRONOLOG_CONF->RPC_BASE_SERVER_PORT);
+        rpc = ChronoLog::Singleton<ChronoLogRPCFactory>::GetInstance()
+                ->GetRPC(CHRONOLOG_CONF->RPC_BASE_SERVER_PORT);
         func_prefix = "ChronoLog";
         switch (CHRONOLOG_CONF->RPC_IMPLEMENTATION) {
```

Comment:
Does MetadataRPCVisor and AdminRPCVisor require separate RPC instances? Can you invoke one RPC instance and use it for both Admin and Metadata Visors? Are you creating two visors? Why not have one visor class that does both Admin and Metadata operations?

Reply:
Yes, there is actually only one RPC instance. All RPCs are invoked via that instance. That is how the current Visor test case works. I just want to split the logic into separate classes so that a simplified Visor can have only admin or metadata APIs.

Reply:
Ok

Reply:
When I run chronovisor_server_test, two RPC instances are created. Will you be making the change for a single RPC instance in this issue or is that going to be a separate issue?